tests/impl.cpp

```diff
@@ -2455,6 +2472,12 @@ result_t test_mm_sad_pu8(const SSE2NEONTestImpl &impl, uint32_t iter)
     return validateUInt16(c, d, 0, 0, 0);
 }
 
+result_t test_mm_set_flush_zero_mode(const SSE2NEONTestImpl &impl,
+                                     uint32_t iter)
+{
+    return TEST_UNIMPL;
```

Comment:
How can we validate this macro?

Reply:
First, we need to create a denormal number, do the addition to the normal floating point numbers,
and check the result.

A denormal number is where the mantissa is non zero, but the exponent value is zero in an IEEE* floating-point representation.
It would be treated as zero when flush zero mode is on.

However, I am not really sure about the exact behaviour and need more investigation.
Hence, I left the validation to the future.

Reply:
Put some considerations in comment(s). Then, add `FIXME` or `TODO`.

Reply:
Done.